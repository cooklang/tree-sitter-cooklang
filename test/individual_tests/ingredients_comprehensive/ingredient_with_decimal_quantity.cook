@olive oil{2.5%tbsp}