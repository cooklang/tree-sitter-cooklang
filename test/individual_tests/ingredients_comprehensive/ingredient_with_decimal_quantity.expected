(recipe
  (step
    (ingredient
      name: (ingredient_text)
      (amount
        (quantity)
        (units)))))